app/models.py

```diff
@@ -2109,6 +2109,10 @@ def serializable_after(self, data):
             data['createdAt'] = data['created_at']
 
         data['signed_agreements'] = self.signed_agreements()
+
+        if self.supplier:
+            data['assessed_domains'] = self.supplier.assessed_domains
```

Comment:
Yes it does - this is a convenience to prevent having to write code on supplier route to copy domains to application